java/source/create.rst

```diff
@@ -0,0 +1,134 @@
+======================
+Creating arrow objects
+======================
+
+A vector is the basic unit in the java arrow columnar format.
```

Comment:
```suggestion
A vector is the basic unit in the Arrow Java library.
```

Reply:
I'm not going to go through and mark all of these, but let's please make sure to capitalize names properly.

Reply:
Changed